src/layouts/buttons.js

```diff
@@ -0,0 +1,50 @@
+import Button from '../components/button/Button';
+
+export class Buttons {
+  constructor() {
+    this.buttons = null;
+  }
+
+  render() {
+    if (!this.buttons) {
+      this.buttons = document.createElement('section');
+    }
+    this.buttons.innerHTML = '';
+    this.buttons.classList.add('buttons__container');
+    this.buttons.style.gridArea = 'buttons';
+    this.buttons.style.margin = '1rem 0';
+    this.buttons.style.alignSelf = 'start';
+    this.buttons.style.minWidth = '100%';
+    this.buttons.style.gap = '10px';
+    this.buttons.style.display = 'flex';
+    this.buttons.style.justifyContent = 'space-between';
+
+    // const ButtonSettings = new Button({ label: 'Settings', type: 'default', width: 'full' });
```

Comment:
delete comments from the code

Reply:
During development the comments can be useful to the others and can be removed  when going to production. As I said in Trello's comment I didn't know where to place this button and it was important to me to show that I didn't forget about this button :)